standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStore.java

```diff
@@ -240,6 +247,20 @@
   private static ZooKeeperHiveHelper zooKeeperHelper = null;
   private static String msHost = null;
 
+  static final String OPERATION_FIELD_NAME = "operation";
```

Comment:
Some documentation of these constants will help improve the readability.

Reply:
Noted, I'll add some comments